olive/model.py

```diff
@@ -247,6 +219,53 @@ def get_execution_providers(self, device: Device):
 
         return eps if eps else available_providers
 
+    def get_io_config(self):
```

Comment:
>get_io_config [](http://example.com/codeflow?start=8&length=13)

should the logic be part of load_model and let prepre_session use the loaded model?

Reply:
Updated as per our discussion offline